### ISSUE TITLE
Help req: spectrum-colorpicker directive not rendering in DOM

### ISSUE THREAD

**Issue by benoror** [Textalk/angular-schema-form-colorpicker#1]
Hello! I'm trying to render this control but for some reason the directive is not rendered in the DOM:

```
<div ng-init="defaultSpectrumOptions = { showInput: true, showAlpha: true, allowEmpty: true, showPalette: true, preferredFormat: form.colorFormat || 'rgb', palette: [['#fce94f', '#fcaf3e', '#e9b96e'], ['#8ae234', '#729fcf', '#ad7fa8'], ['#ef2929', '#888a85', '#deface']] }">
    <spectrum-colorpicker ng-model="model['Color']" format="form.colorFormat || 'rgb'" style="background-color: white" type="color" schema-validate="form" options="form.spectrumOptions || defaultSpectrumOptions" class="ng-pristine ng-untouched ng-valid ng-valid-schema">
    </spectrum-colorpicker>
</div>
```

form:

```
{
key: "Color",
colorFormat: "hex",
spectrumOptions: {preferredFormat: "hex"},
type: "colorpicker"
}
```

schema:

```
Color: {
    format: "color",
    title: "Color",
    type: "string"
}
```

model:

```
Color: "#00e944"
```

**Comment by Dervisevic** [comment]
So i tested your form/schema in the examples thing and a colorpicker popped out. So the first thing that comes to mind is if you've follower the [install instructions](https://github.com/Textalk/angular-schema-form-colorpicker#installation) to a t, as wrong order or something missing could be the case. Also check if you have any errors in the console, as that might indicate what's wrong.

**Comment by benoror** [comment]
Thanks for your feedback @Dervisevic!

The issue was indeed related to the script's loading order, as I misunderstood the README.

> **TL;DR;** spectrum-related stuff should be loaded **_before**_ angular-schema-form

Actually I'm developing a fairly complex app, therefore I use bower to handled my dependencies, and spectrum was placed after schema-form.

Can you elaborate on the issue, so I can't prevent collisions with other add-ons/controls in the future?

Thanks again and great work!

**Comment by Dervisevic** [comment]
I'm not quite sure what you mean by elaborate on the issue, but i assume you're asking why they have to be loaded in that order.

Let's look at the list again.
1. jQuery
2. Angular
3. The Spectrum source files (see the GitHub page for documentation)
4. The Spectrum-angular source files (see the GitHub page for documentation)
5. The spectrum CSS
6. Translation files for whatever language you want to use (optional)
7. Angular Schema Form
8. The Angular Schema Form Colorpicker files (this project)

Let's look from the bottom and up, the most important thing is to load the projects file last as it depends on angular schema form, and ASF has no knowledge of any addons. Before you can load this projects files you have to load spectrum, so that comes before. And spectrum needs jQuery, so that has to be loaded before. So the order above isn't set in stone, as some things can be shifted, it's merely a suggestion for success :)

**Comment by benoror** [comment]
Cool, thanks for the clarification @Dervisevic !
